index.html

```diff
@@ -0,0 +1,127 @@
+  </code>
+  <h1><span class=get>GET</span>/leagues/great-league/leads/{slug}</h1>
+  <code>
+    <span class=desc>Method</span>: <span class=get>GET</span>
+    <span class=desc>Route</span>: https://placeholder.url/leagues/great-league/leads
+    <span class=desc>Headers</span>: Authorization: Bearer YOUR_API_KEY_HERE
+    <span class=desc>Description</span>: Returns the number of times the Pokémon with the associated slug has been encountered in Great League, and the ratings the encounters happened.
+  </code>
+  <h1><span class=post>POST</span>/leagues/great-league/leads</h1>
+  <code>
+    <span class=desc>Method</span>: <span class=post>POST</span>
+    <span class=desc>Route</span>: https://placeholder.url/leagues/great-league/leads  
+    <span class=desc>Headers</span>:
+    Content-Type: application/json
+    Authorization: Bearer YOUR_API_KEY_HERE
+
+    <span class=desc>Body</span>:
+    { "name": "Charizard", "slug": "charizard", "rating": 1500 }
+
+    <span class=desc>Description</span>: Creates a new Great League lead entry from the data provided in the body.
+  </code>
+  <h1><span class=delete>DELETE</span>/leagues/great-league/leads/{slug}</h1>
+  <code>
+    <span class=desc>Method</span>: <span class=delete>DELETE</span>
+    <span class=desc>Route</span>: https://placeholder.url/leagues/great-league/leads/{slug}
+    <span class=desc>Headers</span>: Authorization: Bearer YOUR_API_KEY_HERE
+    <span class=desc>Description</span>: Deletes the Pokémon Great League lead entry associated with the provided slug.
+  </code>
+  <h1 id="ultra-league"><span class=get>GET</span>/leagues/ultra-league/leads</h1>
+  <code>
+    <span class=desc>Method</span>: <span class=get>GET</span>
+    <span class=desc>Route</span>: https://placeholder.url/leagues/ultra-league/leads
+    <span class=desc>Headers</span>: Authorization: Bearer YOUR_API_KEY_HERE
+    <span class=desc>Description</span>: Returns a list of all lead Pokémon players have encountered in Ultra League.
+  </code>
+  <h1><span class=get>GET</span>/leagues/ultra-league/leads/{slug}</h1>
+  <code>
+    <span class=desc>Method</span>: <span class=get>GET</span>
+    <span class=desc>Route</span>: https://placeholder.url/leagues/ultra-league/leads
+    <span class=desc>Headers</span>: Authorization: Bearer YOUR_API_KEY_HERE
+    <span class=desc>Description</span>: Returns the number of times the Pokémon with the associated slug has been encountered in Ultra League, and the ratings the encounters happened.
+  </code>
+  <h1><span class=post>POST</span>/leagues/ultra-league/leads</h1>
+  <code>
+    <span class=desc>Method</span>: <span class=post>POST</span>
+    <span class=desc>Route</span>: https://placeholder.url/leagues/ultra-league/leads  
+    <span class=desc>Headers</span>:
+    Content-Type: application/json
+    Authorization: Bearer YOUR_API_KEY_HERE
+
+    <span class=desc>Body</span>:
+    { "name": "Charizard", "slug": "charizard", "rating": 1500 }
```

Comment:
This may be my misunderstanding of your data, but would Charizard show up as a lead many times? Would this data potentially hold:

```JSON
{ "name": "Charizard", "slug": "charizard", "rating": 1500 }
{ "name": "Charizard", "slug": "charizard", "rating": 1800 }
```

If so then your delete routes will need to change. Unless you would want to accept "charizard" and wipe out all instances. If there are multiple instances of a slug then your deletes will need to use IDs and your POST will need to return that information so the user knows the ID to use.